modules/core/src/main/scala/ch/epfl/bluebrain/nexus/kg/core/resources/RepresentationId.scala

```diff
@@ -13,7 +13,7 @@ import ch.epfl.bluebrain.nexus.kg.core.UriOps._
   * @param schemaId   the unique identifier (inside this project) of the schema which constrains this resource. TODO: Change the type to a [IRI] or [Id]
   */
 final case class RepresentationId(projectRef: String, resourceId: String, schemaId: String) {
-  def persId: String = s"${projectRef.hashCode}${resourceId.hashCode}${schemaId.hashCode}"
+  def persId: String = s"${projectRef.hashCode.abs}${resourceId.hashCode.abs}${schemaId.hashCode.abs}"
```

Comment:
The current implementation of persistence id is vulnerable to collisions and it makes it difficult to inspect the primary store directly. I would suggest we use something like `{projectRef}_{resourceId}_{schemaId}`.